src/main/kotlin/net/atos/client/smartdocuments/SmartDocumentsClient.kt

```diff
@@ -47,4 +41,12 @@ interface SmartDocumentsClient {
         @HeaderParam("Authorization") authenticationToken: String,
         @HeaderParam("Username") userName: String
     ): SmartDocumentsTemplatesResponse
+
+    @GET
+    @Path("smartdocuments/result/show")
+    @Consumes("text/html; charset=UTF-8")
```

Comment:
Is it not enough to do:

```suggestion
    @Consumes(MediaType.TEXT_HTML)
```
?

Is UTF8 not the default in our endpoints anyway?

Reply:
> By default if no Content-Type header is present in a part, "text/plain; charset=us-ascii" is used as fallback. This is the value defined by the MIME RFC.

Reply:
Yes, I know I think. We had issues with this in multi-part forms I believe where UTF8 was not uses because of this..